src/elements/DropdownMulti.vue

```diff
@@ -168,6 +168,13 @@ export default {
       type: Boolean,
       default: false,
     },
+    /**
+     * Autoclose the dropdown
+     */
+    autoclose: {
```

Comment:
I would suggest making autoclose camel case autoClose. Everything is good though man!